tests/keras_tests/layer_tests/base_keras_layer_test.py

```diff
@@ -105,15 +106,16 @@ def create_networks(self):
         networks = []
         for i, layer in enumerate(layers):
             inputs = [Input(shape=s[1:]) for s in self.get_input_shapes()]
-            if self.is_inputs_a_list:
+            if isinstance(layer, partial) and layer.func is tf.image.combined_non_max_suppression:
```

Comment:
What's the meaning of checking "isinstance(layer, partial)"? why would it be of type "partial"? (If this is something that is sewed for the test then at least add a comment explaining it, or maybe try to find a way to make the flow more explicit)

Reply:
it's the standard way to mimic tf functions as Keras layers. used extensively in the Keras layer tests.

Reply:
The usage of partial is fine, I was referring for the way that you identify the layer in the test by using `isinstance(..., partial)`, which seems cryptic and unclear.
So, either identify the layer in a different way or at least add a comment explaining what you are expecting to find here